gateway/src/api/latest.rs

```diff
@@ -284,9 +287,16 @@ async fn delete_project(
                     .and_then(|x| x.parse::<semver::Version>().ok())
             })
         })
-        .unwrap_or(lowest_restartable_version.clone());
-
-    if !project_deletable || version.le(&lowest_restartable_version) {
+        // Defaulting to a version that introduced a breaking change.
+        // This was the last one that introduced it at the present
+        // moment.
+        .unwrap_or(semver::Version::new(0, 39, 0));
```

Comment:
non-blocking: The image tag failing to be read should always lead to a restart, so this might as well be (0, 0, 0)